### PR TITLE
Removed wrong information

### PR DESCRIPTION
**Pull request by joshblakeley**
Don't know who added this grpc proxying information but its not correct.

### CODE REVIEW

**Comment by joshblakeley** [comment]
Infact @marksou can you remove this whole page

File: tyk-docs/content/plugins/supported-languages/rich-plugins/grpc/grpc.md [hunk 1]
```diff
@@ -9,26 +9,3 @@ url: "/plugins/supported-languages/rich-plugins/grpc"
...
   -  "/plugins/rich-plugins/grpc"
 ---
-## What is gRPC?
-From the [gRPC documentation](http://www.grpc.io/faq/ ):
-
-> gRPC is a modern, open source remote procedure call (RPC) framework that can run anywhere. It enables client and server applications to communicate transparently, and makes it easier to build connected systems.
-
-### How can I use gRPC with Tyk?
-
-Using Tyk with your gRPC client and server is very easy. Since gRPC uses HTTP/2, you need to enable it by setting `enable_http2=true` for `Downstream-Tyk` and `proxy_enable_http2=true` for `Tyk-Upstream` connections. You also need to set your `listen_path` in your API definitions.
```
Comment (sedkis) [inline tyk-docs/content/plugins/supported-languages/rich-plugins/grpc/grpc.md:19]: @joshblakeley  I wonderr if any of these fields are required for gRPC plugins ?  `enable_http2` for example

Comment (matiasinsaurralde) [reply tyk-docs/content/plugins/supported-languages/rich-plugins/grpc/grpc.md:19]: Not required for plugins.

**Review by marksou — APPROVED** [review]
👍

**Pull request merged**